javascript/src/views/keypad-universe.html

```diff
@@ -0,0 +1,8 @@
+<ul class="nav nav-tabs">
```

Comment:
This duplication feels very repetitive and prone to errors, can the menu generation not be automated or run through a preprocessor?